book/03-git-branching/sections/basic-branching-and-merging.asc

```diff
@@ -19,13 +19,14 @@ You'll do the following:
 ==== Basic Branching
 
 (((branches, basic workflow)))
-First, let's say you're working on your project and have a couple of commits already.
+First, let's say you're working on your project and have a couple of commits already
+on the `master` branch.
```

Comment:
Please remove the line break here (notice how the diff isn't highlighting the new words in the sentence).

Reply:
Yup, reverting all those line breaks as we speak.